src/frontend/Komponenter/Behandling/TidligereVedtaksperioder/typer.ts

```diff
@@ -11,12 +11,13 @@ export interface ITidligereInnvilgetVedtak {
     harTidligereBarnetilsyn: boolean;
     harTidligereSkolepenger: boolean;
     periodeHistorikkOvergangsstønad: IGrunnlagsdataPeriodeHistorikk[];
+    periodeHistorikkBarnetilsyn: IGrunnlagsdataPeriodeHistorikk[];
 }
 
 export interface IGrunnlagsdataPeriodeHistorikk {
-    antallMåneder: number;
-    antallMånederUtenBeløp: number;
-    vedtaksperiodeType: EPeriodetype;
+    antallMåneder?: number;
+    antallMånederUtenBeløp?: number;
+    vedtaksperiodeType?: EPeriodetype;
```

Comment:
Nå brukes denne både for OS og BT - kanskje vi kunne kostet på oss en kommentar på felter som bare er for OS?

Reply:
Vi får jo sannsynligvis enda en inn her: overlappper med OS som bare er for BT.

Reply:
Kan vurdere å skille på grunnlagsdata OS og BT? Men det blir jo endringer i mange filer (men vi slipper ?)